_includes/share-mentor.html

```diff
@@ -0,0 +1,29 @@
+{% assign base_url = 'https://womencodingcommunity.com' %} 
```

Comment:
You could create `base_url` in _config.yml and re-use like this `site.base_url`

Reply:
@dricazenck  done